src/components/document/resize-panel-divider.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import DragThumbnailIcon from "../../assets/drag-thumb-icon.svg";
+import LeftRightDragIcon from "../../assets/left-right-drag.svg";
+import "./resize-panel-divider.scss";
+
+interface IProps {
+  isResourceExpanded: boolean;
+  dividerPosition: number;
+  onExpandWorkspace: () => void;
+  onExpandResources: () => void;
+}
+
+export const ResizePanelDivider: React.FC <IProps> =
+  ({isResourceExpanded, dividerPosition, onExpandWorkspace, onExpandResources}) => {
+    const dividerLeftOffset = 22;
+    const tabWidth = 45;
+    const dividerPositionStyle = dividerPosition  === 0
+                                  ? {left: dividerLeftOffset}
+                                  : dividerPosition === 50
+                                      ? {left: `calc(${dividerPosition}% - ${dividerLeftOffset}px)`}
+                                      : {left: `calc(${dividerPosition}% - ${tabWidth}px - ${dividerLeftOffset}px)`};
```

Comment:
If you reverse the polarity of the second ternary, you won't have to change it when dragging is implemented:
```typescript
    const dividerPositionStyle = dividerPosition  === 0
                                  ? {left: dividerLeftOffset}
                                  : dividerPosition === 100
                                      ? {left: `calc(${dividerPosition}% - ${tabWidth}px - ${dividerLeftOffset}px)`}
                                      : {left: `calc(${dividerPosition}% - ${dividerLeftOffset}px)`};
```